### ISSUE TITLE
Audio: spurious audio packets

### ISSUE THREAD

**Issue by nor1bie** [sm0svx/svxlink#501]
This maybe already reported. 
As I remember it's been on the table for many years.

**The issue:** 
Recently installed SVXLINK (after a move) onto a _Intel D525MW_ single board computer. The computer is running _Debian 10_. The interface is one of my own, **modified to mimic the S54S interface for DTMF decoding** with a tone control circuit. The svxlink version is from the Debian repository (apt install). Haven't had a chance to check the version from the repository, but is certainly not the same as from latest source code. I tested my hardware, with different firmware using the EchoLink Sysop software and it does what I expected it to do. Unfortunately, Windows isn't that great, plus I don't have a license to run.
 
1. When starting svxlink, for some reason **REMOTETRX** is started and taking possession of the audio device. Stopping remotetrx will allow svxlink to use the audio device. This is probably not a big issue. Perhaps in init.d, I'll resolve this later.

2. After temporarily resolving the **audio device** issue, there are numerous Errors pertaining to spurious audio packets. I believe the error will in most cases not allow the system to function properly. It might simply drop a connection before it can resume. 

3. I am aware that the parrot module is different, however using the parrot works beautifully. Audio is great, no distortions.

I have been experimenting with audio volumes in and out for the **EchoLinkModule.** As the audio level decreases the spurious packets are less or disappear. At the same time, the audio level is so low, that there is hardly anything heard. Also, connecting to the EchoLink Test Server, does not produce the sign on message. Lowering the playback volume, may or may not produce the desired effects. It is merely a random effect that trends to the not working side.
For example: I connect to EchoTest, nothing works. Disconnect and reconnect, no preample, my audio comes back. The routine repeated, without any changes, it won't work. Then again next time it works, in some instance, even the preample actually makes it through.
I ran out of things that I can do to resolve the issue myself. There aren't many references to this particular problem on the web either. Mostly Raspberry user report this behavior.

Perhaps, I do not understand the meaning of the error message. What causes the spurious packet? What does this message actually mean and what does it relate to.
Anyone that can lend me a hand is appreciated.
KJ6ZD, 73

**Comment by nor1bie** [comment]
Update:
I went ahead and took Linux and SVXLINK off. Instead I installed Windows with the windows version of echolink. This installation seems to work flawless. Even though I tried for probably a week or more to get SVXLINK to work, I simply failed with this task. Perhaps, the software from the repositories just don't want to work? I tried Debian 10, with the version from repository. I tried Ubuntu server 20.04 with the version from repository. I did not try to compile the latest version from github. At this point I do not know if this would work. I noticed that with previous install, it appears to work, however nobody can connect to the link, we tried the Phone version of Echolink, which simply timed out, yet a connection to the Test-Server did work?
KJ6ZD, 73
